Mathlib/RepresentationTheory/GroupCohomology/Resolution.lean

```diff
@@ -162,94 +159,6 @@ end Basis
 
 end groupCohomology.resolution
 
-namespace Rep
-
-variable (n) [Group G] (A : Rep k G)
-
-open groupCohomology.resolution
-
-/-- Given a `k`-linear `G`-representation `A`, the set of representation morphisms
-`Hom(k[Gⁿ⁺¹], A)` is `k`-linearly isomorphic to the set of functions `Gⁿ → A`. -/
-noncomputable def diagonalHomEquiv :
-    (Rep.diagonal k G (n + 1) ⟶ A) ≃ₗ[k] (Fin n → G) → A :=
-  Linear.homCongr k
-        ((diagonalSuccIsoTensorTrivial k G n).trans
-          ((Representation.ofMulAction k G G).repOfTprodIso 1))
-        (Iso.refl _) ≪≫ₗ
-      (Rep.MonoidalClosed.linearHomEquivComm _ _ _ ≪≫ₗ Rep.leftRegularHomEquiv _) ≪≫ₗ
-    (Finsupp.llift A k k (Fin n → G)).symm
```

Comment:
I was going to deprecate this and its API because I no longer use it to set up group cohomology and it's easy to derive from existing definitions. But for now I've just moved it to `RepresentationTheory.Rep` and golfed the API, just in case.